go.mod

```diff
@@ -3,3 +3,8 @@ module go.yaml.in/yaml/v2
 go 1.15
```

Comment:
You, probably, need to change version of go as well, because the retract directive was added in v16: https://go.dev/ref/mod#go-mod-file-retract

Reply:
Good catch

It would require some tests maybe.

While what you said is right, I feel like.

If the lib is used with Go 1.15, the user won't get warned, you are right. But the go get will fail anyway because package name won't satistify its import path.

If the lib is used with Go 1.16+, the retract will be parsed and it will be OK.

Also, the retract thing is about informing about the status on pkg.go.dev

But good catch.